index-rebalances/utils/rebalanceLib.ts

```diff
@@ -37,6 +37,8 @@ export function getRebalanceInputs(
   };
 }
 
+// DPI allocations are a function of token market cap. Some tokens (ex: uni) are "flexibly capped"
+// at 25% of the index even though their natural weight would be higher.
```

Comment:
Might be worth rephrasing this so it doesn't sound like UNI is being singled out/chosen to be "flexibly capped".

`Any token which would comprise more than 25% of the total value in the Set is "flexibly capped" at 25%.`